Feed/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def feed(request):
+    return render(request, 'Feed/index.html')
```

Comment:
If we are changing the name of the app from "Feed" to "feed", I think the folder in the templates should be "feed" also.

Reply:
We're not changing the name of the app to feed,
We're changing the name of the method to feed only.

Reply:
lol in that case LGTM

Reply:
@AlonShakaroff Please read [this](https://learndjango.com/tutorials/django-best-practices-projects-vs-apps) regarding the app naming convention.

Reply:
@natifridman Ok, so if our main model in that app is Post, we should change the app name to posts?

Reply:
yes

Reply:
I think it will be best if @OmerCS8 will change the name of the app in his PR, 
because he already based his work on my PR and it will prevent complications so I am closing this PR.